transitions/multi_form_alerts.js

```diff
@@ -208,5 +214,6 @@ module.exports = {
     doc.type === 'data_record' &&
     !transitionUtils.hasRun(doc, NAME)
   ),
-  onMatch: onMatch
+  onMatch: onMatch,
+  init: validateConfig
```

Comment:
I don't want to add more on to your plate necessarily, but do you think this is a good opportunity to do a once-over of existing transitions and see if there are quick `init`s we can add to those? Either here on via a raised ticket.